templates/base.html

```diff
@@ -0,0 +1,12 @@
+<head>
+  <title>Taxi Service</title>
+  {% load static %}
+  <link rel="stylesheet" href="{% static 'css/styles.css' %}">
+</head>
+<body>
+  {% block sidebar %}
+    {% include "includes/sidebar.html" %}
+  {% endblock %}
+  {% block content %}
+  {% endblock %}
+</body>
```

Comment:
Add blank line to the end of every html file

Reply:
add blank line to the end of file

Reply:
tap Enter at the end of last line - it will add a new line at the end of file no need to at <br> here )
![image](https://user-images.githubusercontent.com/103201010/211338862-1f8650e6-5630-4648-b742-2bb398b27b8c.png)

Reply:
U've just delete a` <br>` but don't add a blank line 
just blank line like this 
![image](https://user-images.githubusercontent.com/103201010/211342584-0f1db34e-5872-44aa-9110-edcc8d9ba5dc.png)